server/server.js

```diff
@@ -27,6 +27,18 @@ app.get('/overview/:product_id', async (req, res) => {
     data.overviewInfo = overviewInfo.dataValues;
     data.materialsInfo = materialsInfo.data;
     res.send(data);
+  } catch (err) {
+    const overviewInfo = await db.overview.findByPk(productNum, { include: [db.icons] });
+    data.overviewInfo = overviewInfo.dataValues;
+    res.send(data);
+  }
+});
+
+app.get('/icons/:product_id', async (req, res) => {
```

Comment:
Do these handle the case of searching for a product and finding nothing, but there was no error thrown? or does an error always get thrown if you can't find something in the db?

Reply:
An error gets thrown if you cant find something in the DB. I need to also add a case for that and just display some text from starting state or something. I will update that soon but for now I am going to merge this and deploy it.